modules/images/webp-uploads/load.php

```diff
@@ -4,41 +4,414 @@
  * Description: Uses WebP as the default format for new JPEG image uploads if the server supports it.
  * Experimental: No
  *
+ * @since   1.0.0
  * @package performance-lab
- * @since 1.0.0
  */
 
+add_filter( 'wp_generate_attachment_metadata', 'webp_uploads_create_images_with_additional_mime_types', 10, 3 );
```

Comment:
Generating the additional mime types immediately might fail because it will double execution time, right? I can test this locally, but I think this might break uploads for some users.

Can we instead add an immediately one-time executing cron callback (`wp_schedule_single_event`)? that way the images will get generated on the next callback as part of a separate request?

Reply:
Agree I'm going to explore this as part of enchantment on top of this PR [as you describe](https://github.com/WordPress/performance/pull/154#issuecomment-1040833914) has become quite large at this point.

Reply:
Actually I just updated this PR to handle this case.